docs/cookbook/building-navigation.md

```diff
@@ -0,0 +1,292 @@
+:::
+
+## Generating site navigation from the page tree
+
+One of the defining features of pages in Apostrophe is that they exist in a hierarchy. Even if we added all pages on a website as siblings (the same hierarchy level) they would still all be children of the home page. This structure is expressed through the [page tree](/guide/pages.md#connecting-pages-with-page-tree-navigation).
+
+The Apostrophe demo site is a small example of common page tree structure. Under the home page there are two **top-level pages**. One of those pages, "Editor Experience," has three additional **child pages**, making it their **parent page**. These pages can all be rearranged through drag-and-drop interaction.
+
+![Screenshot of the Apostrophe demo page structure in the page manager UI](/images/recipes/demo-page-tree.png)
+
+Apostrophe templates have access to this page tree using `data.home._children`, an array of top-level page data objects. Templates also have access to the children of the page a visitor is on, but since we're focused on the main site navigation, we want to use the home page's children. Home page children do not change as visitors move around the website.
+
+As explained on the [pages guide](/guide/pages.md#connecting-pages-with-page-tree-navigation), we can loop over `data.home._children` and print the pages' URLs and titles in a list. Once you add some classes and CSS, this can work great as site navigation. And since it is based on the page tree structure **it will automatically stay up to date as editors update page content**.
+
+<AposCodeBlock>
+  ```django
+  <header>
+    <nav>
+      <ul>
+        {# 👇 Referencing `data.home._children` and looping over them. #}
+        {% for page in data.home._children %}
+          <li>
+            <a href="{{ page._url }}">{{ page.title }}</a>
+          </li>
+        {% endfor %}
+      </ul>
+    </nav>
+  </header>
+  ```
+  <template v-slot:caption>
+    views/layout.html
+  </template>
+</AposCodeBlock>
+
+### Change page tree data in templates
+
+The example above is a very simple site navigation. It includes only the top-level pages and the basic data available for each page. In some cases you may want different data from the page tree for your navigation. The direct way to do this is to alter the **query builders** that the core page module uses to get that data.
+
+The *default* query builder options for the page module look like this:
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    options: {
+      builders: {
+        children: true,
+        ancestors: {
+          children: true
+        }
+      }
+    }
+  }
+  ```
+  <template v-slot:caption>
+    modules/@apostrophecms/page/index.js
+  </template>
+</AposCodeBlock>
+
+For example, **to include two levels of pages in the nav** we would add to the `builders` option that ancestors (such as the home page) should include two levels of child pages (`depth: 2`).
```

Comment:
People will take this literally. Should say it makes it *possible* to include two levels of pages in the nav, and then show how to do that at template level.

Reply:
I'm trying not to overwhelm too much with examples, but I'll add a note about the additional layer of `_children` array in `data.home._children` items. And fix the misleading bold bit here.